src/main/java/com/iwaconsolti/school/config/SchoolPopulatedConfig.java

```diff
@@ -0,0 +1,96 @@
+package com.iwaconsolti.school.config;
+
+import com.iwaconsolti.school.model.Course;
+import com.iwaconsolti.school.model.Grade;
+import com.iwaconsolti.school.model.schools.GerardoInstitute;
+import com.iwaconsolti.school.model.School;
+import com.iwaconsolti.school.model.schools.ZetCollege;
+import com.iwaconsolti.school.model.Student;
+import com.iwaconsolti.school.service.SchoolService;
+import com.iwaconsolti.school.service.SchoolServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.beans.factory.annotation.Value;
+
+
+@Profile("populated")
+@Configuration
+@Slf4j
+public class SchoolPopulatedConfig {
+    @Value("${school.score.limit:100}")
+    private int limitGrade;
+
+    @Bean
+    public School gerardoInstitute() {
+        return new GerardoInstitute();
+    }
+
+    @Bean
+    public School zetCollege() {
+        return new ZetCollege();
+    }
+
+    @Bean("gerardoService")
+    public SchoolService gerardoServicePopulated() {
+        School gerardoInstitute = gerardoInstitute();
+        SchoolServiceImpl schoolService = new SchoolServiceImpl(gerardoInstitute, limitGrade);
+
+        Student student1 = new Student(1, "Juan", "Perez", 20);
+        Student student2 = new Student(2, "Ana", "Gomez", 22);
+
+        schoolService.addStudent(student1);
+        schoolService.addStudent(student2);
+
+        Course course1 = new Course(1, "Matemáticas", "Prof. Ramirez");
+        Course course2 = new Course(2, "Historia", "Prof. López");
+
+        schoolService.addCourse(course1);
+        schoolService.addCourse(course2);
+
+        Grade grade1 = new Grade(1, 85, student1, course1);
+        Grade grade2 = new Grade(2, 90, student1, course2);
+        Grade grade3 = new Grade(3, 78, student2, course1);
+        Grade grade4 = new Grade(4, 88, student2, course2);
+
+        schoolService.addGrade(grade1);
+        schoolService.addGrade(grade2);
+        schoolService.addGrade(grade3);
+        schoolService.addGrade(grade4);
+
+        log.info("gerardoServicePopulated initialized");
+        return schoolService;
+    }
+
+    @Bean("zetService")
+    public SchoolService zetServicePopulated() {
```

Comment:
Methods should start with a verb describing the action.
Change:
- zetServicePopulated()
To:
- createZetServicePopulated()